code/game/turfs/open/floor.dm

```diff
@@ -28,12 +28,18 @@
 
 
 /turf/open/floor/Initialize(mapload)
-	if (!broken_states)
-		broken_states = string_list(list("damaged1", "damaged2", "damaged3", "damaged4", "damaged5"))
+	if (broken_states)
+		stack_trace("broken_states defined at the object level for [type], move it to setup_broken_states()")
 	else
-		broken_states = string_list(broken_states)
-	if(burnt_states)
-		burnt_states = string_list(burnt_states)
+		var/list/new_broken_states = setup_broken_states()
+		if(new_broken_states)
+			broken_states = string_list(new_broken_states)
```

Comment:
```suggestion
		broken_states = string_list(setup_broken_states())
```
There should always be `broken_states`, unlike `burnt_states`, right?

Reply:
yes